source/write/gridfs.txt

```diff
@@ -401,22 +413,27 @@ Delete Files
 
 Use the ``MongoDB\GridFS\Bucket::delete()`` method to remove a file's collection
 document and associated chunks from your bucket. This effectively deletes the file.
-You must specify the file by its ``_id`` field rather than its file name.
+Pass the ``_id`` value of the file you want to delete as a parameter to the
+``delete()`` method.
 
 The following example shows you how to delete a file by referencing its ``_id`` field:
 
 .. literalinclude:: /includes/write/gridfs.php
    :language: php
    :dedent:
-   :start-after: start-delete-files
-   :end-before: end-delete-files
+   :start-after: start-delete-files-id
+   :end-before: end-delete-files-id
 
-.. note:: File Revisions
+Alternatively, you can use the ``MongoDB\GridFS\Bucket::deleteByName()`` method
+to delete a file and all its file revisions. Pass the ``filename`` value of the
```

Comment:
Text above said "rename a GridFS file and all...", but you omitted "GridFS" here. Should both of these sentences be consistent? I don't have any preference between them.

Reply:
Added GridFS!